_posts/0000-01-02-KelinTRS.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
![smart-cat-incopy-533029935](https://user-images.githubusercontent.com/98394962/150978094-ebc6ee70-e092-445b-a518-4cbcd356cda1.jpg)
The _only_ right logic.